src/test/java/tests/api/Case/CaseTest.java

```diff
@@ -0,0 +1,85 @@
+package tests.api.Case;
+
+import baseEntities.BaseApiTest;
+import models.Case;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CaseTest extends BaseApiTest {
+
+//    Это CRUD тест. Необходимо запускать весь класс, так как все тесты зависимы методом dependsOnMethods.
+//    Я добавил в этот тест добавление кейса, а так же удаление проекта чтобы не плодить проекты в testRail.
+//    При запуске тестов, будет выводиться информация о создании проекта и создании тест кейса с их id-ишниками.
+
+    @Test
+    public void addCase() {
+        projectId = projectAdapter.addProject();
+
+        sectionId1 = sectionAdapter.addSection1(projectId);
+
+        expectedCase = Case.builder()
+                .title("Case Test Number 1")
+                .sectionId(sectionId1)
+                .templateId(2)
+                .typeId(6)
+                .priorityId(2)
+                .estimate("5")
+                .build();
+
+        caseId = caseAdapter.addCase(sectionId1);
+    }
+
+    @Test(dependsOnMethods = "addCase")
+    public void getCase() {
+        expectedCase = Case.builder()
+                .title("Case Test Number 1")
+                .sectionId(sectionId1)
+                .templateId(2)
+                .typeId(6)
+                .priorityId(2)
+                .estimate("5m")
+                .build();
+
+        expectedCase.setId(caseId);
+
+        Case actualCase = caseAdapter.getCase(caseId);
+
+        Assert.assertEquals(actualCase, expectedCase);
+    }
+
+    @Test(dependsOnMethods = "getCase")
+    public void updateCase() {
+        sectionId2 = sectionAdapter.addSection2(projectId);
+
+        Case updatedCase = Case.builder()
```

Comment:
{
    "error": "Field :suite_id is not a valid test suite."
}

java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <400>.